source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2614,6 +2653,14 @@ export class HTML5MediaPlayer extends MediaPlayer {
             Utils.appendChild(this._mediaElement, renderedSource);
         }
 
+        for (const captionSource of this.owner.captionSources) {
+            if (captionSource.mimeType == "vtt") {
```

Comment:
In UWP you've included:
![image](https://user-images.githubusercontent.com/21356912/161826929-a3c04be6-a1c1-4fb8-b755-a2902cc4d32c.png)
maybe `srt` should be supported in JS too?:)

Reply:
JS [does not support `srt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track)!

Reply:
Then should we really support it in UWP?

Reply:
> JS [does not support `srt`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track)!

my apologies :) had no idea :)

Reply:
> Then should we really support it in UWP?

It's a good question :)
I'm pretty certain we can easily convert `srt` to `vtt` with a simple routine (so we can support both in JS) but not certain if it's worth doing :)

Reply:
Each platform should declare support for every common sidecar caption format it understands.

If a card author has a variety of caption formats at their disposal, they should add them all as caption sources. 

Each platform renderer should select a caption source that it supports from the author-supplied options (and should be encouraged to select the richest format it can).

This also enables scenarios where someone is interested in an accessible media story, but isn't interested in going cross-platform (e.g. IT app dev has a .wmv and a .srt file on a Sharepoint and wants to author a card to show the video in their Windows-only UWP LOB app -- they shouldn't have to transcode the video, convert the captions file, and reupload to Sharepoint in order to render a card). We should, if possible, document which formats are supported on each platform, but I wouldn't block this PR on that.

Reply:
Also, we should definitely avoid doing any file format conversions if at all possible -- just as a general rule.

Reply:
Okay, so uwp will support both `vtt` and `srt`, JS will only support `vtt`. Resolving this thread